### PR TITLE
[specific ci=Group11-Upgrade] Correct certPath output on vic-machine create

### PR DESCRIPTION
**Pull request by gigawhitlocks**
This fixes the displayed certificate path after a create. 

@emlin do I need to update `inspect` to show this information as well, in order to close issue #4683 ? Or is `inspect`'s output fine? Or do we have another issue for updating `inspect`?

### CODE REVIEW

**Comment by emlin** [comment]
@gigawhitlocks #4490 is for the inspect error

File: lib/install/management/inspect.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func (d *Dispatcher) GetDockerAPICommand(conf *config.VirtualContainerHostConfig
 			}
 
 			dEnv = append(dEnv, "DOCKER_TLS_VERIFY=1")
-			info, err := os.Stat(conf.ExecutorConfig.Name)
+			info, err := os.Stat(certpath)
```
Comment (emlin) [inline lib/install/management/inspect.go]: what happens if user didn't specify certpath, but used custom cert and key files?

Comment (gigawhitlocks) [inline lib/install/management/inspect.go]: That's a good question. What is the correct output if they're in different locations?

Comment (emlin) [inline lib/install/management/inspect.go]: I feels like user does not have to specify cert-path if custom key/cert files are specified, so here I think the output should allow empty certpath case.

Comment (gigawhitlocks) [inline lib/install/management/inspect.go]: Looks like --key and --cert are only for input. --cert-path specifies input (if certs exist) AND output, so this change should work fine with --key and --cert

**Pull request closed without merging**